### DISCUSSION TITLE
How to disable field type inferencing?

### DISCUSSION THREAD

**Discussion by aborruso** [nushell/nushell#3483] [Q&A]
Hi,
if my input is this CSV 

```
exampleField
047
067
```

and I open it, I have field type inferencing, text to integer casting

```
───┬──────────────
 # │ exampleField
───┼──────────────
 0 │           47
 1 │           67
───┴──────────────
```

How to open it in nushell to have the same cell values I have as input?

Thank you

**Comment by kubouch** [comment]
You can do `open --raw file.csv | lines` and work your way from there.

The CSV conversion seems to automatically convert the values. It might be a good feature to add the option to disable type inference and return everything as strings.

**Reply by aborruso** [reply]
@kubouch I know, but I do not want to use it as raw.

I would like to open it, and apply to it nushell commands and subcommands. But if a value is changed from `047` to `47`, I can't proceed, because that value is a code.

I think it would be very important to have some "no inferencing" mode, or some way, some option, to set the format for one or more field during opening.

**Reply by kubouch** [reply]
I see, I don't think there is currently a way. You can file a feature request issue for this, I think it would be a valuable thing to add.

I would imagine it as a flag you would give to `from csv` or more generally to `open`, for example.

**Reply by kubouch** [reply]
There is `split column` command you could use as a workaround -- should be good enough for simple CSV parsing.

**Reply by aborruso** [reply]
> There is `split column` command you could use as a workaround -- should be good enough for simple CSV parsing.

I know, but CSV is a "complicated" format,  and if I have some separator in the cells, probably I will have a wrong split.

I think it's better to integrate no inferencing mode option as feature.

```
exampleField,another
047,"My name, andy"
067,"Yes, we can"
```

**Reply by kubouch** [reply]
I see, yeah, it's not really a good solution. It was just the only workaround I could think of, but eventually we should have the no-inference feature.

**Reply by aborruso** [reply]
> You can file a feature request issue for this

Here is the issue https://github.com/nushell/nushell/issues/3485
